components/TestimoniesTable/TestimoniesTable.js

```diff
@@ -15,14 +15,19 @@ const TestimonyRow = ({ testimony }) => {
   const senatorId = testimony.senatorId
   const representativeId = testimony.representativeId
 
+  const x = usePublicProfile(testimony.authorUid)
+  const authorPublic = x.result?.public
+  const loading = x.loading
+  const error = x.error
+
   return (
     <tr>
       <td>{testimony.billId}</td>
       <td>{testimony.position}</td>
       <td>
         {testimony.authorDisplayName == null ? (
-          "(blank)"
-        ) : (
+          <>(blank)</>
+        ) : authorPublic ? (
           <Button
             variant="primary"
             onClick={() =>
```

Comment:
We can actually render the Button as an `<a>` element, which can make loading a bit faster. [We do that on the home page](https://github.com/codeforboston/advocacy-maps/blob/76e42828ee55acedcaa5e4051b329add46e61eaa/pages/index.tsx#L37). We should do that whenever we use the button as a link

```
            <Wrap href={`/publicprofile?id=${testimony.authorUid}`}>
              <Button>{testimony.authorDisplayName}</Button>
            </Wrap>
```

Reply:
nice!  I updated.